### DISCUSSION TITLE
Update for v0.10.0

### DISCUSSION THREAD

**Discussion by update-freak** [MrAlucardDante/baikal-docker-hass#10] [General]
Hi developer,

will this repo also be updated to the latest Baikal version v0.10.0?
(https://github.com/sabre-io/Baikal/releases)

**Comment by MrAlucardDante** [comment]
Hi. 

Sure. But my image is based on ckulka's image. So v10 will be out once they release it, which they did 15 hours ago. 

I'll look into it today, because it has Postgres support, which I was looking forward to try.

**Comment by MrAlucardDante** [comment]
> Hi developer,
> 
> 
> 
> will this repo also be updated to the latest Baikal version v0.10.0?
> 
> (https://github.com/sabre-io/Baikal/releases)

Just in case, I released 0.10.1 a few days ago with postgres support. Enjoy!
